spinalcordtoolbox/scripts/sct_apply_transfo.py

```diff
@@ -32,29 +32,26 @@ def get_parser():
         description='Apply transformations. This function is a wrapper for antsApplyTransforms (ANTs).'
     )
 
-    mandatory = parser.add_argument_group("MANDATORY ARGUMENTS")
+    mandatory = parser.mandatory_arggroup
     mandatory.add_argument(
         "-i",
-        required=True,
-        help='Input image. Example: t2.nii.gz',
+        help='Input image. Example: `t2.nii.gz`',
```

Comment:
Thank you so much for making these changes! All of the syntax here gets parsed as Markdown when displayed in the docs (e.g. [`sct_apply_transfo`](https://spinalcordtoolbox--4819.org.readthedocs.build/4819/user_section/command-line/sct_apply_transfo.html#mandatory-arguments)), so this formatting work goes beyond just CLI convention!